config.go

```diff
@@ -80,7 +80,7 @@ type ZapProperties struct {
 }
 
 func (cfg *Config) buildOptions(errSink zapcore.WriteSyncer) []zap.Option {
-	opts := []zap.Option{zap.ErrorOutput(errSink)}
+	opts := []zap.Option{zap.ErrorOutput(errSink), zap.AddCallerSkip(1)}
```

Comment:
`AddCallerSkip(1)`  might need to change because the call stack level chage?

I'm not sure, if the call stack is still correct, that's OK.